tests/unit/sagemaker/model/test_neo.py

```diff
@@ -304,26 +300,7 @@ def test_compile_with_framework_version_16(session):
         job_name="compile-model",
     )
 
-    assert "1.6" in model.image_uri
-
-
-@patch("sagemaker.session.Session")
-def test_compile_validates_framework_version(session):
-    session.return_value.boto_region_name = REGION
-
-    model = _create_model()
-    with pytest.raises(ValueError) as e:
-        model.compile(
-            target_instance_family="ml_c4",
-            input_shape={"data": [1, 3, 1024, 1024]},
-            output_path="s3://output",
-            role="role",
-            framework="pytorch",
-            framework_version="1.6.1",
-            job_name="compile-model",
-        )
-
-    assert "Unsupported neo-pytorch version: 1.6.1." in str(e)
```

Comment:
any specific reason we are removing negative test case

Reply:
Same as the previous reply.
Since the uri is fetched from Neo API now, the API response will put `inference uri` as `None` if there is not a proper inference image.

Reply:
lets add a test case to make sure it is  happening